src/app/lib/layoutGrid.js

```diff
@@ -303,3 +303,28 @@ export const gridContainerSmallCss = css`
     max-width: initial;
   }
 `;
+
+export const customGridItemMedium = css`
+  ${gelGridMargin}
+
+  grid-column: 1 / span 6;
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    grid-column: 1 / span 5;
+    max-width: ${fiveOfSixColumnsMaxWidthScaleable};
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    grid-column: 3 / span 5;
+    max-width: ${fiveOfSixColumnsMaxWidthGroup4};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    grid-column: ${props => props.gridColumnStart} / span
+      ${props => specifiedOrMaximum(props.gridSpan, 22)};
+    max-width: ${tenOfTwelveColumnsMaxWidthGroup5};
+    max-height: 1.375rem;
```

Comment:
```suggestion
    max-height: 0;
```
I think we should change this value to 0. I went to take a screenshot in storybook of the behaviour that Drew and Rhys mentioned and you can see there's a little gap where the grid item has popped out.
![Current grid pop out behaviour in storybook](https://user-images.githubusercontent.com/19362408/59829988-980cbf00-9336-11e9-9639-8b729662f662.png)

But if you change the `max-height` to 0 we get this instead:
![Possible future pop out grid item behaviour](https://user-images.githubusercontent.com/19362408/59830017-a78c0800-9336-11e9-8a4c-c3d8ebc59f0e.png)

Thank goodness for those coloured blocks?